src/lib/sw_engine/tvgSwRasterAvx.h

```diff
@@ -24,6 +24,29 @@
 
 #include <immintrin.h>
 
+#define N_32BITS_IN_128REG 4
+
+static inline __m128i ALPHA_BLEND_AVX(__m128i c, __m128i a)
```

Comment:
"AVX" in the naming  is not quite necessary since this implementation is in the AVX local space.

Reply:
I tried to use the naming convention from the neon version of the file. I'll change it